osu.Game.Tournament/Configuration/TournamentStorageManager.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Configuration;
+using osu.Framework.Platform;
+
+namespace osu.Game.Tournament.Configuration
+{
+    public class TournamentStorageManager : IniConfigManager<StorageConfig>
+    {
+        protected override string Filename => "tournament.ini";
+
+        public TournamentStorageManager(Storage storage)
+            : base(storage)
+        {
+        }
+
+        protected override void InitialiseDefaults()
+        {
+            base.InitialiseDefaults();
+            Set(StorageConfig.CurrentTournament, string.Empty);
```

Comment:
Is this even necessary if it's going to be overwritten upon first execution of `TournamentStorage` anyway?